app/templates/_footer_categories.html

```diff
@@ -24,7 +24,7 @@
         <a href="https://www.gov.uk/guidance/the-g-cloud-framework-on-the-digital-marketplace">G-Cloud framework</a>
       </li>
       <li>
-        <a href="{{ url_for('main.suppliers_list_by_prefix') }}">G-Cloud supplier A–Z</a>
+        <a href="/g-cloud/suppliers">G-Cloud supplier A–Z</a>
```

Comment:
I guess we're not using an external blueprint for this because there aren't any query parameters?